Server/controllers/index.js

```diff
@@ -1,7 +1,7 @@
 const router = require('express').Router();
 
-router.get('/', (req, res) => {
-  res.send('Server is Running');
-});
+const { getUserBillType } = require('./Bill');
+
+router.get('/api/user/:userId/home', getUserBillType);
 
```

Comment:
What the point of having `id` in URL, if its expected to be taken from user cookies?

Reply:
I thank the endpoint must change to `/api/home`